packages/issuer/src/blockchain-facade/RequestIssue.ts

```diff
@@ -0,0 +1,99 @@
+import { Configuration, BlockchainDataModelEntity, Timestamp } from '@energyweb/utils-general';
+import { PublicIssuer } from '..';
+
+export interface IRequestIssueOnChainProperties  {
+    id: string;
+    owner: string;
+    fromTime: Timestamp,
+    toTime: Timestamp,
+    deviceId: string;
+    approved: boolean;
+}
+
+export class Entity extends BlockchainDataModelEntity.Entity implements IRequestIssueOnChainProperties {
+
+    owner: string;
+    fromTime: Timestamp;
+    toTime: Timestamp;
+    deviceId: string;
+    approved: boolean;
+    initialized: boolean;
+
+    constructor(id: string, configuration: Configuration.Entity) {
```

Comment:
Constructor could be removed if `initialized` declaration would declare its value as well.

Reply:
Not really, the constructor also calls `super(id, configuration)`;